tox.ini

```diff
@@ -3,30 +3,33 @@ envlist =
     py{27,py,34,35,36,37}{,-subunit},coverage,docs
 
 [testenv]
-deps =
-    .[test]
-    subunit: .[subunit]
+extras =
+    test
+    subunit: subunit
 commands =
     python setup.py -q test -q
 
 [testenv:coverage]
 usedevelop = true
 basepython =
-    python2.7
+    python3
```

Comment:
I didn't know you could do that (specify a major but not a minor).

Reply:
https://www.python.org/dev/peps/pep-0394/ says there ought to be a `python2` and a `python3` in $PATH on Unix-like systems.

It's possible not all distros actually implement that convention (*cough*Arch*cough*).